src/main/kotlin/io/github/mojira/arisa/infrastructure/HelperMessages.kt

```diff
@@ -116,36 +121,48 @@ object HelperMessageService {
     fun getMessageWithBotSignature(project: String, key: String, filledText: String? = null, lang: String = "en") =
         getMessage(project, listOf(key, "i-am-a-bot"), listOf(filledText), lang)
 
-    fun setHelperMessages(json: String) = data.fromJSON(json).rightIfNotNull {
-        Error("Couldn't deserialize helper messages from setHelperMessages()")
-    }
-
-    fun updateHelperMessages(file: File) = fetchHelperMessages().fold(
-        {
-            if (file.exists()) {
-                data.fromJSON(file.readText()).rightIfNotNull {
-                    Error("Couldn't deserialize saved helper messages")
-                }
-            } else {
-                it.left()
-            }
-        },
-        { inputStream ->
-            inputStream.use {
-                file.writeText(data.toJSON())
-                data.fromStream(it).rightIfNotNull {
-                    Error("Couldn't deserialize downloaded helper messages")
+    fun setHelperMessages(json: String) = data.fromJSON(json)
+        ?: throw IOException("Couldn't deserialize helper messages from setHelperMessages()")
```

Comment:
Maybe that is not even needed, it appears Klaxon throws exceptions for invalid JSON, so maybe result is only `null` when input is JSON `null`?